src/app/layouts/base-layout/components/header/nav-item/nav-item.module.scss

```diff
@@ -52,6 +52,11 @@
       width: 16px;
       height: 17px;
```

Comment:
maybe make height here 16px too, as i can see it doesnt affect visually, but in code it looks like 'whats going on here?" when these magic numbers appear and differ from each other

Reply:
I remove it at all, thank you